jobs/ci-kubernetes-e2e-gce-canary.env

```diff
@@ -0,0 +1,4 @@
+# Everything else is the same as the other job it is testing
+BOOTSTRAP_METADATA_CANARY_IMAGE=true
+KUBEKINS_TIMEOUT=50m
```

Comment:
Done. It is running fewer tests so I want to redefine this here (and it also needs to exist to pass unit tests).